### PR TITLE
Allow custom host on aleph dev

### PR DESCRIPTION
**Pull request by hazae41**

### CODE REVIEW

File: cli/dev.ts [hunk 1]
```diff
@@ -9,17 +9,21 @@ Usage:
 if the <dir> is empty, the current directory will be used.
 
 Options:
+    -h, --host       A hos to start the Aleph.js app, default is localhost
```
Comment (shadowtime2000) [inline cli/dev.ts]: Should be `-hn, --host`. Also typo with `A host`. See [cli/start.ts#L11](https://github.com/alephjs/aleph.js/blob/master/cli/start.ts#L11)

File: cli/dev.ts [hunk 2]
```diff
@@ -9,17 +9,21 @@ Usage:
...
 export default async function (appDir: string, options: Record<string, string | boolean>) {
+    const host = String(options.h || options.host || 'localhost')
     const port = parseInt(String(options.p || options.port || '8080'))
+    
     if (isNaN(port) || port <= 0 || !Number.isInteger(port)) {
         log.error(`invalid port '${options.port || options.p}'`)
         Deno.exit(1)
     }
-    start(appDir, 'localhost', port, true, Boolean(options.r || options.reload))
+
```
Comment (shadowtime2000) [inline cli/dev.ts:27]: Any reason for this extra line?

File: cli/dev.ts [hunk 3]
```diff
@@ -9,17 +9,21 @@ Usage:
 if the <dir> is empty, the current directory will be used.
 
 Options:
+    -h, --host       A host to start the Aleph.js app, default is localhost
```
Comment (shadowtime2000) [inline cli/dev.ts:12]: Use `-hn, --host` instead.

Comment (hazae41) [reply cli/dev.ts]: Done

File: cli/dev.ts (hunk 2 above)
Comment (hazae41) [reply cli/dev.ts]: Just for readability; anyway, removed

File: cli/dev.ts [hunk 4]
```diff
@@ -9,17 +9,20 @@ Usage:
...
     -p, --port       A port number to start the Aleph.js app, default is 8080
     -L, --log-level  Set log level [possible values: debug, info]
     -r, --reload     Reload source code cache
     -h, --help       Prints help message
 `
 
 export default async function (appDir: string, options: Record<string, string | boolean>) {
+    const host = String(options.hn || options.host || 'localhost')
     const port = parseInt(String(options.p || options.port || '8080'))
+
```
Comment (shadowtime2000) [inline cli/dev.ts:22]: please remove this extra line

Comment (hazae41) [reply cli/dev.ts]: Done

**Review by shadowtime2000 — APPROVED** [review]
lgtm

**Pull request merged**